.github/workflows/eden_gcp.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: EdenGCP
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows: ["publish"]
+    branches: [main]
+    types:
+      - completed
+# yamllint disable rule:line-length
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        hv: ["kvm", "xen"]
+    steps:
+      - name: get eden
+        uses: actions/checkout@v2
+        with:
+          repository: 'lf-edge/eden'
+      - name: fetch or build eve
```

Comment:
Since you're now doing the right thing with triggering it on publish -- we can now completely get rid of this step -- Eden will fetch the docker artifact by itself -- there's no need for it to be explicit.